x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx

```diff
@@ -196,7 +196,7 @@ export class TOCEntryActionsPopover extends Component<Props, State> {
                 defaultMessage:
                   'Edit features only supported for document layers without clustering, term joins, time filtering, or global search.',
```

Comment:
Can you change this message to be "To enable edit features, select 'Use vector tiles' in 'Scaling'."